Scroller/EditableWrapper.js

```diff
@@ -364,14 +368,12 @@ const EditableWrapper = (props) => {
 
 	useEffect(() => {
 		// Calculate the item width once
-		if (!mutableRef.current.itemWidth) {
-			const item = wrapperRef.current?.children[0];
-			const neighbor = item.nextElementSibling || item.previousElementSibling;
-			mutableRef.current.itemWidth = Math.abs(item.offsetLeft - neighbor?.offsetLeft);
-			mutableRef.current.centeredOffset = centered ? item.getBoundingClientRect().x : 0;
-			wrapperRef.current?.style.setProperty('--item-width', mutableRef.current.itemWidth + 'px');
-		}
-	}, [centered]); // TODO: Need dataSize dependency for centeredOffset
+		const item = wrapperRef.current?.children[0];
```

Comment:
I think this useEffect() function will be replaced with function that is implemented in https://github.com/enactjs/sandstone/pull/1233 PR.